src/layouts/layout.tsx

```diff
@@ -1,25 +1,26 @@
 import React, { useEffect, useState } from "react";
 import type { ReactNode } from "react";
-import { ArrowForwardIosOutlined } from "@mui/icons-material";
 import { useRouter } from "next/router";
 import { useAppDispatch, useAppSelector } from "@/redux/hooks";
 import { getProfile } from "@/redux/profileSlice";
-import { signOut } from "next-auth/react";
-import { Header } from "./Header";
+import { getListSkill } from "@/redux/skillSlice";
+import { ArrowForwardIosOutlined } from "@mui/icons-material";
 import { Modal } from "@mui/material";
 import { Footer } from "./Footer";
+import { Header } from "./Header";
+;
 
 type IHeaderProps = {
   children: ReactNode;
 };
+
 const Layout = (props: IHeaderProps) => {
-  const data = useAppSelector((state) => state.ProfileReducer.userInfo);
```

Comment:
we need to create the selector function on each reducer to get a piece of state. Example: in the profileReducer we will have the function like 
`getUserInfoSelector = (state) => state.ProfileReducer.userInfo`

Reply:
Done